### ISSUE TITLE
最新战报，知乎相关问题已被删除。该问题一度占据知乎热搜第一

### ISSUE THREAD

**Issue by GoatGirl98** [ShiArthur03/ShiArthur03#1014]
只能说这并不是一个意外的结果，在简中互联网混的大伙也没有必要装外宾。作为吃瓜群众的我们能做的也只有持续关注了，但愿于老师回复的 [“电机系已经构成了调查专班，调查结论择机公布”](https://github.com/ShiArthur03/ShiArthur03/issues/112) 还能兑现吧。github 这边应该不会有大手伸过来（除非举报人这边真的被爆了）。哪怕真的是造谣，我个人也希望这个项目活到该事件的大结局前夕再似。

**Comment by KaysinBIT** [comment]
gmail注册一个邮箱又不需要国内手机号，没法查

**Comment by GoatGirl98** [comment]
> gmail注册一个邮箱又不需要国内手机号，没法查

不太懂，不过不少拿蓝鸟等app实施诈骗等违法活动的人，如果人还在境内的话，六扇门应该也是有其他手段能定位到的。校方也能对整个实验室的在读学生进行高强度调查，总还是能查到举报人的。只能说但愿目前举报人平安吧。

**Comment by KaysinBIT** [comment]
诈骗的一般都会有资金流动，主要是这玩意没有经济冲突，凭借账号定位确实很困难。

**Comment by HansongYan666** [comment]
压力给到了，我看改成华清大学了

**Comment by KaysinBIT** [comment]
> 压力给到了，我看改成华清大学了

估计是咨询过律师？这属于写科幻小说了

**Comment by sweetning0809** [comment]
> > gmail注册一个邮箱又不需要国内手机号，没法查
> 
> 不太懂，不过不少拿蓝鸟等app实施诈骗等违法活动的人，如果人还在境内的话，六扇门应该也是有其他手段能定位到的。校方也能对整个实验室的在读学生进行高强度调查，总还是能查到举报人的。只能说但愿目前举报人平安吧。

应该还好吧 作者2小时前还更新了

**Comment by starfish233e** [comment]
> > gmail注册一个邮箱又不需要国内手机号，没法查
> 
> 不太懂，不过不少拿蓝鸟等app实施诈骗等违法活动的人，如果人还在境内的话，六扇门应该也是有其他手段能定位到的。校方也能对整个实验室的在读学生进行高强度调查，总还是能查到举报人的。只能说但愿目前举报人平安吧。

社工比技术手段威胁大多了，他这明显是同组爆破，代码都能拿到。真要查就那么几个人，谁干的大家心里都有数😉

**Comment by GoatGirl98** [comment]
> > > gmail注册一个邮箱又不需要国内手机号，没法查
> > 
> > 
> > 不太懂，不过不少拿蓝鸟等app实施诈骗等违法活动的人，如果人还在境内的话，六扇门应该也是有其他手段能定位到的。校方也能对整个实验室的在读学生进行高强度调查，总还是能查到举报人的。只能说但愿目前举报人平安吧。
> 
> 社工比技术手段威胁大多了，他这明显是同组爆破，代码都能拿到。真要查就那么几个人，谁干的大家心里都有数😉

确实。不过被举报人的博士学位论文最后写的学术成果当中，不少合作者都已经毕业了。一个比较理想的情况就是他的合作者平时留心眼对代码和同组其他人的博士学位论文进行了留档，并且已经毕业脱离学校了。这种情况下，至少校方施加的压力是不用担心的。

**Comment by HFUT-CHEATER** [comment]
> > gmail注册一个邮箱又不需要国内手机号，没法查
> 
> 不太懂，不过不少拿蓝鸟等app实施诈骗等违法活动的人，如果人还在境内的话，六扇门应该也是有其他手段能定位到的。校方也能对整个实验室的在读学生进行高强度调查，总还是能查到举报人的。只能说但愿目前举报人平安吧。

只要是真的，被查也不怕

**Comment by HFUT-CHEATER** [comment]
> [HansongYan666](/HansongYan666)

https://github.com/HFUT-CHEATER/HFUTCheaterCollection 这个直接发不出来

**Comment by KimLay233** [comment]
好的收到！请再探再报！

**Comment by HansongYan666** [comment]
> > [HansongYan666](/HansongYan666)
> 
> https://github.com/HFUT-CHEATER/HFUTCheaterCollection 这个直接发不出来

@HFUT-CHEATER  替换一些内容再发知乎，如果这件事上面重视，趁这个热度写举报信。

**Comment by august-li-alt** [comment]
> > [HansongYan666](/HansongYan666)
> 
> https://github.com/HFUT-CHEATER/HFUTCheaterCollection 这个直接发不出来

这个内容有点太哈人了，把人家身份证号都贴出来了
